diagnostic.md

```diff
@@ -73,14 +97,27 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --include --request "POST" "http://www.example.com/donuts" \
+  --header "Content-Type: application/json" \
+  --data $'{
+    "donuts": {
+      "name": "French Cruller",
+      "price": "$0.99"
+    }
+  }'
 ```
 
 Write an AJAX request to create a single donut on the server using JSON. Please
 do not use `data = getFormFields(form)` instead write out the data object.
 
 ```js
-let createDonut = /* your answer here */;
+const createDonut = function (form) {
+  return $.ajax({
+    url: app.host + '/donuts/',
+    method: 'POST',
+    data: getFormFields(form),
```

Comment:
READ " do not use `data = getFormFields(form)` instead write out the data object"